### PR TITLE
removes 'feature_rds' flag

### PR DESCRIPTION
**Pull request by lsh-0**

### CODE REVIEW

File: src/Search/Kernel.php [hunk 1]
```diff
@@ -434,8 +433,7 @@ public function dependencies(Application $app)
                 $app['console.gearman.task_driver'],
                 $app['elastic.client.write'],
                 $app['validator'],
-                $app['logger'],
-                $app['config']['feature_rds'] ? $app['config']['rds_articles'] : []
```
Comment (nlisgo) [inline src/Search/Kernel.php:438]: The condition needs to be removed but the parameter is still expected. I will followup with a commit.

**Review by nlisgo — CHANGES_REQUESTED** [review]

**Review by nlisgo — APPROVED** [review]

**Comment by nlisgo** [comment]
@lsh-0 It has been a long time since CI has been green for any PR's

I will leave you to merge this in.

**Comment by lsh-0** [comment]
> I will leave you to merge this in.

I'm more inclined to close it now to be honest.

It came up in Giorgio's PR fix, but now that RDS articles are properly integrated and are simply regular articles, do we need special handling for them in search at all? Can we just strip out all special handling of RDS articles? I can do this myself if it's the right thing to do.

**Comment by lsh-0** [comment]
> @lsh-0 It has been a long time since CI has been green for any PR's

green now

**Pull request merged**